code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,318 @@
+/datum/adminticket
```

Comment:
please add a comment explaining briefly what each var is for if possible,especially vars like..uckey?and uID,which i don't understand